### PR TITLE
October 2024 changes

### PR DESCRIPTION
**Pull request by ivanruzak**
https://hjelpesenter.finn.no/hc/no/articles/21153547277330-Mobility-integration-info
https://hjelpesenter.finn.no/hc/no/articles/21153765520530-Migration-of-Mobility-business-customers-to-FINN-Bedriftssenter

### CODE REVIEW

File: src/AdTypes/McXml.php [hunk 1]
```diff
@@ -46,18 +46,9 @@ public function __construct($partner_id, $provider)
...
         $this->contactBody->removeAttribute('PHONESALESRESERVATION');
-        $this->contactBody->removeChild($this->contactFaxBody);
         $this->contactBody->removeChild($this->contactURLBody);
     }
 
-    public function setContactFax($fax)
-    {
-    }
-
-    public function setContactUrl($url)
```
Comment (eiriksm) [inline src/AdTypes/McXml.php:57]: Maybe we should keep this one? We don't want it to be possible to set contact url if its not present in the Dtd

File: src/AdTypes/ThreshingMachineXml.php [hunk 2]
```diff
@@ -43,6 +43,18 @@ public function __set($name, $value)
...
     {
-      // Empty on purpse. There is no segment.
+      // Empty on purpose. There is no segment.
+    }
+
+    public function addEquipment($equipment_items)
+    {
+        // Remove initial empty tag.
+        $this->customTags['TRESHING_MACHINE_EQUIPMENT']->remove();
+        foreach ($equipment_items as $equipment_value) {
```
Comment (eiriksm) [inline src/AdTypes/ThreshingMachineXml.php:58]: Would be super nice to add a test for this. Would you know how to? Do you want me to help?

**Review by eiriksm — CHANGES_REQUESTED** [review]

File: src/AdTypes/ThreshingMachineXml.php (hunk 2 above)
Comment (ivanruzak) [reply src/AdTypes/ThreshingMachineXml.php:58]: It's a bit urgent PR, so probably it will be better, if you'll help :sweat_smile:

**Review by eiriksm — APPROVED** [review]
Great. If tests are passing lets do it

**Comment by eiriksm** [comment]
@ivanruzak seems like the method ::remove is not there for php 7.4

which unfortunately is relevant for LA I guess. Could you look at it?

**Comment by ivanruzak** [comment]
Yes, just wanted to say the same. I'll check now.

**Comment by ivanruzak** [comment]
@eiriksm 
Can we go now?

**Review by eiriksm — APPROVED** [review]
:shipit: :shipit: :shipit:

**Pull request merged**